v2/bind.go

```diff
@@ -15,18 +17,39 @@ type bindRequestBody struct {
 	Context      map[string]interface{} `json:"context,omitempty"`
 }
 
+type bindSuccessResponseBody struct {
+	Credentials     map[string]interface{} `json:"credentials"`
+	SyslogDrainURL  *string                `json:"syslog_drain_url"`
+	RouteServiceURL *string                `json:"route_service_url"`
+	VolumeMounts    []interface{}          `json:"volume_mounts"`
+	Operation       *string                `json:"operation"`
```

Comment:
optional fields should be omitempty, right

Reply:
Since we're strictly using this struct to unmarshal into, I don't think it's needed. Or does omitempty affect unmarshaling as well?